modules/standard/BitOps.chpl

```diff
@@ -246,7 +257,7 @@ private module BitOps_internal {
     return BitOps_internal.ctz(x:uint(bits)):int(bits);
   }
 
-  inline proc popcount(x: uint(?bits)) {
+  inline proc popCount(x: uint(?bits)) {
```

Comment:
Since this and the later function are internal, technically you didn't have to adjust them (but I'm glad you did and wouldn't undo it) :)